model/business-process-new/base.js

```diff
@@ -79,7 +79,88 @@ module.exports = memoize(function (db/*, options*/) {
 			if (this.manager) result.push(_observe(this.manager._email));
 
 			return result;
-		} }
+		} },
+		toWebServiceJSON: {
+			value: function (ignore) {
+				var dataFields = {}, result, db;
+				db = this.database;
+				result = {
+					id: this.__id__,
+					service: {
+						code: this.constructor.__id__
+					},
+					submittedTimestamp:
+						this._isSubmitted.lastModified ?
+								Math.floor(this._isSubmitted.lastModified / 1000) : null,
+					processingSteps: {},
+					request: {
+						registrations: [],
+						documentUploads: [],
+						costs: [],
+						payments: [],
+						certificates: [],
+						data: {}
+					}
+				};
+
+				this.processingSteps.map.forEach(function self(processingStep) {
+					if (db.ProcessingStepGroup && processingStep instanceof db.ProcessingStepGroup) {
+						processingStep.steps.applicable.forEach(self);
+						return;
+					}
+					result.processingSteps[processingStep.key] = processingStep.toWebServiceJSON();
+				});
+
+				this.registrations.requested.forEach(function (reg) {
+					result.request.registrations.push({
+						code: reg.key
+					});
+				});
+
+				this.requirementUploads.applicable.forEach(function (upload) {
```

Comment:
Is it sth more then a gut feeling?  I believe that applicable collection works fine, I checked it for ELS and got requirementUploads of nested objects.